Composer/packages/client/src/recoilModel/dispatchers/project.ts

```diff
@@ -543,16 +544,20 @@ export const projectDispatcher = () => {
 
   const fetchReadMe = useRecoilCallback((callbackHelpers: CallbackInterface) => async (moduleName: string) => {
     try {
+      callbackHelpers.set(fetchReadMePendingState, true);
       const response = await httpClient.get(`/assets/templateReadme`, {
         params: { moduleName: encodeURIComponent(moduleName) },
       });
 
       if (response.data) {
+        callbackHelpers.set(fetchReadMePendingState, false);
```

Comment:
why here and again below?

Reply:
could be in a finally block since it happens in all three cases

Reply:
Ah silly me forgetting about the beauty if finally{}. Handled this.